tests/testUtils.py

```diff
@@ -244,6 +312,8 @@ def waitForObj(lam, timeout=None, sleepTime=3, reporter=None):
 
     @staticmethod
     def waitForBool(lam, timeout=None, sleepTime=3, reporter=None):
```

Comment:
changed waitForObj(...) to waitForTruth (since python docs refer to __bool__ as returning the truth value) and replacing all calls to waitForBool with that